File: Java/01.Java-Basic/06.ConditionalStatementsAdvancedExercise/src/OperationsBetweenNumbers.java
import java.util.Scanner;

public class OperationsBetweenNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int first = Integer.parseInt(scanner.nextLine());
        int second = Integer.parseInt(scanner.nextLine());
        String operation = scanner.nextLine();

        double result = 0;
        String type = "";
        switch (operation) {
            case "+":
            case "-":
            case "*":
                if (operation.equals("+")) {
                    result = first + second;
                } else if (operation.equals("-")) {
                    result = first - second;
                } else {
                    result = first * second;
                }

                type = result % 2 == 0 ? "even" : "odd";
                System.out.printf("%d %s %d = %.0f - %s", first, operation, second, result, type);
                break;

            case "/":
            case "%":
                if (second != 0) {
                    if (operation.equals("/")) {
                        result = first / (second * 1.00);
                        System.out.printf("%d %s %d = %.2f", first, operation, second, result);
                    } else if (operation.equals("%")) {
                        result  = first % second;
                        System.out.printf("%d %s %d = %.0f", first, operation, second, result);
                    }

                } else {
                    System.out.printf("Cannot divide %d by zero", first);
                }
                break;

            default:
                break;
        }
    }
}

File: JavaScript/01.JavaScriptBasic/06Nested-Conditional-Statements-Еxercise/src/OperationsBetweenNumbers.js
function expression(params) {
    let a = Number(params[0]);
    let b = Number(params[1]);
    let operator = params[2];

    let result = 0;
    let type = "";

    if (operator === "+") {
        result = a + b;
        type = result % 2 === 0 ? "even" : "odd";
        console.log(`${a} + ${b} = ${result} - ${type}`);
    } else if (operator === "-") {
        result = a - b;
        type = result % 2 === 0 ? "even" : "odd";
        console.log(`${a} - ${b} = ${result} - ${type}`);
    } else if (operator === "*") {
        result = a * b;
        type = result % 2 === 0 ? "even" : "odd";
        console.log(`${a} * ${b} = ${result} - ${type}`);
    } else if (operator === "/") {
        if (b === 0) {
            console.log(`Cannot divide ${a} by zero`);
        } else {
            result = a / b;
            console.log(`${a} / ${b} = ${result.toFixed(2)}`);
        }
    } else if (operator === "%") {
        if (b === 0) {
            console.log(`Cannot divide ${a} by zero`);
        } else {
            result = a % b;
            console.log(`${a} % ${b} = ${result}`);
        }
    }
}

expression(["10","12","+"]);
expression(["123","12","/"]);
expression(["112","0","/"]);
expression(["10","1","-"]);	
expression(["10","3","%"]);
expression(["10","0","%"]);